packages/@uppy/provider-views/src/ProviderView/User.tsx

```diff
@@ -1,4 +1,4 @@
-import { h, Fragment } from 'preact'
+import { h } from 'preact'
```

Comment:
```suggestion
import { h, Fragment } from 'preact'
```

Reply:
It still needs to be in scope, even after #4953?

Reply:
Of course, for the same reason we need `h`. It looks like the e2e is not correctly configured though